02/app.js

```diff
@@ -1,2 +1,15 @@
 const x = 5;
-let result = 0;
\ No newline at end of file
+let result = 0;
+
+for (let i = 0; i <= x; i++) {
+    result = result + i
+}
+console.log(result)
+
+// while loop
+let result2 = 0;
+
+while(result2 !== 15 ) {
+    result2++
```

Comment:
Niestety to jest za mało - tutaj nie sumujemy tylko zwiększamy `result2` o jeden. Dlatego pewnie w warunku napisałeś 15. Tutaj potrzebujemy dodatkowej zmiennej (tak jak w `for` było to `i`), aby przechowywać informacje o kolejnej liczbie, a `result2` to wynik kolejnej operacji. Podpowiedź: powinno być coś na zasadzie `result2 = result2 + j` oraz wew. `while` czegoś brakuje.